src/shaders/collision_circle.fragment.glsl

```diff
@@ -23,7 +23,7 @@ void main() {
 
     float extrude_scale_length = length(v_extrude_scale);
     float extrude_length = length(v_extrude) * extrude_scale_length;
-    float stroke_width = 3.0;
+    float stroke_width = 12.0 * extrude_scale_length;
```

Comment:
@ChrisLoer The viewports I was using when we first implemented this were probably closer to 1:1.5–1:2 than to a square, and subsequently looking at this now, the stroke does seem a bit too thin (it looks good on retina, but doesn't render well on non-retina). I tested out `15.0 * extrude_scale_length` here and it looks better to me.